homeassistant/components/plex/__init__.py

```diff
@@ -96,6 +96,14 @@ def _setup_plex(hass, config):
     """Set up Plex from a config entry."""
     server_config = entry.data[PLEX_SERVER_CONFIG]
 
+    if MP_DOMAIN not in entry.options:
```

Comment:
Shouldn't you add tests for this as well?

Reply:
I'd eventually like to add tests for files outside of `config_flow.py`, but I'm not there quite yet.